### ISSUE TITLE
Boss hp plug-in

### ISSUE THREAD

**Issue by Nawwwwwwww** [Pryaxis/Plugins#83]
Can anyone make me a plug-in for my terraria server where the boss hp would not scale even if it is multiplayer.it's ok if the boss hp would be scaled upto the amount where if there is 2 players are there

**Comment by lucasgames8957** [comment]
@Nawwwwwwww, I suggest contacting one of the contributors or you can find a youtube video to modding!
Let me know if this doesn't work.

**Comment by Yinxoyi** [comment]
does terraria already has this feature?

**Comment by lucasgames8957** [comment]
> does terraria already has this feature?

no, and please do not necropost, I will not do anything if you do or don't but I find it annoying.
